src/util/util.ts

```diff
@@ -108,7 +108,9 @@ const evalReg = new RegExp(/\beval\(([^),]*)\)/g);
 
 // hasEval determine whether matcher contains function eval
 function hasEval(s: string): boolean {
-  return evalReg.test(s);
+  const res: boolean = evalReg.test(s);
```

Comment:
Maybe we can use `string.match()` and count the result instead.

Reply:
not sure but that might slow down the process what do you think?